compiler/testData/codegen/box/defaultArguments/useNextParamInLambda.kt

```diff
@@ -0,0 +1,16 @@
+// IGNORE_BACKEND: JS, JS_IR
+
+fun f(
+    f1: () -> String = { f2() },
+    f2: () -> String = { "OK" }
+): String = f1()
+
+fun box(): String {
+    var result = "fail"
+    try {
+        f()
+    } catch (e : Exception) {
+        result = "OK"
+    }
+    return f { result }
```

Comment:
This looks like it does `f(f1 = { result })` while you also probably want to test `f(f2 = { result })`.

Reply:
Seems it's a bit confusing place cause of default arguments, but https://kotlinlang.org/docs/reference/lambdas.html#passing-a-lambda-to-the-last-parameter
so ```f ( { result })``` not equals to ```f { result }```

Reply:
Yeah, this is testing passing to f2. But it is a good point. Let me change the test to check what happens both when passing to f1 and f2 explicitly. Thanks!